public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx

```diff
@@ -68,6 +68,10 @@ export class GeneralSettingsEditView
     return dashboardSceneGraph.getCursorSync(this._dashboard);
   }
 
+  public getLiveNowTimer(): behaviors.LiveNowTimer {
+    return sceneGraph.findObject(this._dashboard, (s) => s instanceof behaviors.LiveNowTimer) as behaviors.LiveNowTimer;
```

Comment:
I really dislike this from TS :( 

We can avoid casting here if we contemplate the behavior could be `undefined` and in `onLiveNowChange` we check if it is an instance of `behaviors.LiveNowTimer`.